spec.emu

```diff
@@ -1021,15 +1027,15 @@ contributors: Nicolò Ribaudo
             </h1>
             <dl class="header">
               <dt>description</dt>
-              <dd></dd>
+              <dd>Collects the direct post-order list of asynchronous unexecuted transitive dependencies, stopping the depth-first search for a branch when an async dependency is found.</dd>
             </dl>
 
             <emu-alg>
               1. If _seen_ is not specified, let _seen_ be a new empty List.
               1. If _seen_ contains _module_, return ~unused~.
               1. Append _module_ to _seen_.
               1. If _module_ is not a Cyclic Module Record, return ~unused~.
-              1. If _module_.[[Status]] is either ~evaluating~ or ~evaluated~, return ~unused~.
+              1. If _module_.[[Status]] is ~evaluating~, ~evaluating-async~ or ~evaluated~, return ~unused~.
```

Comment:
Actually, I'm not sure about `~evaluating-async~` here.

If we have module A that contains TLA, and module B that imports A, and we:
- `import(A)`, so A's status is `~evaluating-async~`
- `import.defer(B)`, GatherAsynchronousTransitiveDependencies should include A because this dynamic import call should not resolve until when A is resolved.

Reply:
Yes it wasn't quire right. If A contains TLA, then B will be in evaluating-async as well. Therefore import.defer(B) should see that B is already evaluating-async and include B itself as the attachment point, where the algorithm should then effectively just import B.

I've added a change to ensure the above.